tiflash/tiflash-configuration.md

```diff
@@ -60,6 +60,12 @@ http_port = tiflash http 服务端口
     data-dir = proxy 数据存储路径
     config = proxy 配置文件路径
     log-file = proxy log 路径
+<<<<<<< HEAD
+=======
+    log-level = proxy log 级别，默认为 "info"
+    status-addr = 拉取 proxy metrics｜status 信息的监听地址
+    advertise-status-addr = 外部访问 status-addr 的地址，不填则默认是 status-addr
+>>>>>>> dee99bf0... update tiflash config: add proxy log level (#5931)
```

Comment:
@solotzg PTAL and help resolve the conflict. Thanks!

Reply:
> @solotzg PTAL and help resolve the conflict. Thanks!

I think we should close this PR because release-3.1 is useless.